skins/cat17/src/app/tests/test_form_components.js

```diff
@@ -315,10 +313,8 @@ test( 'Bank data component adds change handling function to its elements', funct
 
 	formComponents.createBankDataComponent( store, bankDataComponentConfig );
 
-	assertChangeHandlerWasSet( t, bankDataComponentConfig.ibanElement );
+	assertChangeHandlerWasSet( t, bankDataComponentConfig.ibanElement, 2 );
```

Comment:
IBAN field has two change handlers now, I am not entirely sure if that's correct, I assume it's because of the two events we attached to it? Will look into this still.

Reply:
I have tried looking into it but I am really not sure about this change, maybe you have an idea @gbirke @tzhelyazkova?

Reply:
The test cases for the bank data component can be deleted, as it's replaced with a Vue component